projects/angular/src/popover/signpost/API.md

```diff
@@ -0,0 +1,24 @@
+# Signpost-clarity - Style API
+
+## CSS Variables
+
+| CSS Variable Name                       | Description                                                |
+| --------------------------------------- | ---------------------------------------------------------- |
+| --clr-signpost-content-bg-color         | Changes the background color of the description for all signpost elements.
+| --clr-signpost-content-border-color     | Changes the border color of the description for all signpost elements. 
+| --clr-signpost-action-color             | Changes the color of close icon.
+| --clr-signpost-action-hover-color       | Changes the color of close icon on hover.
+| --clr-signpost-border-radius            | Changes the corner radius of the element.
+| --clr-signpost-pointer-border           | Changes the pointer radius of the element.
+| --clr-signpost-pointer-invisible-border | Changes the shape of border to pointer border.
+| --clr-signpost-pointer-psuedo-border    | Changes the color of pointer border.
+## Class names
+
+| Class name         | Description                              |
+| -------------------| ---------------------------------------- |
+| clr-signpost-demo  | Container class for signpost element |
+| signpost-demo      | Class that wraps all signpost elements |
+| signpost-item      | Class contains the signpost element |
+| signpost-positions | The signpost position is specified by Clsss |
+| clr-signpost       | Class is a directional marker that indicates where you are in your path |
```

Comment:
This is not a CSS class name.

Reply:
Hi Kevin, is this about line number 23 , if not please mention line number.
Thank you

Reply:
My comments are about the line I commented on...

Reply:
Updated, As suggested